components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -88,6 +89,10 @@ open class GleanInternalAPI internal constructor () {
         applicationContext: Context,
         configuration: Configuration = Configuration()
     ) {
+        // In certain situations Glean.initialize may be called from a process other than the main
```

Comment:
Can we please file a bug about clarifying the story with Glean and multi process? If somebody tries to record stuff off the main process, thanks to this it will complain about Glean not being initialized. We might want to send a different message or something. Let's take this to a bug :)

Reply:
[Bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1558181) filed